Makefile

```diff
@@ -100,11 +105,11 @@ print-version:
 dist: $(TARFILE)
 	@ls -1 $(TARFILE)
 
-# when building a distribution tarball, call bundler with a 'production' environment
+# when building a distribution tarball, call webpack with a 'production' environment by default
```

Comment:
Please revert the "webpack" word, "bundler" is correct. This isn't using webpack.